packages/schema-blocks/src/core/blocks/BlockInstruction.ts

```diff
@@ -51,17 +54,17 @@ export default abstract class BlockInstruction extends Instruction {
 	 * @param props The props.
 	 * @param i     The number the rendered element is of its parent.
 	 *
-	 * @returns The sidebar element to render.
+	 * @returns {ReactElement | string} The sidebar element to render.
```

Comment:
It now returns a ReactElement right? I think you can remove the `| string` part?
```suggestion
	 * @returns {ReactElement} The sidebar element to render.
```

Reply:
this breaks inherited blocks that DO return a string. so, not for now, unfortunately.